### ISSUE TITLE
Attendee social links fail minimum target size

### ISSUE THREAD

**Issue by amberhinds** [wpa11yday/conference#224]
The `[attendees]` shortcode outputs social media links that fail target size minimums. The largest one, X, is only 20x20px and there is no padding or margin between the link next to it.

<img width="359" height="185" alt="Image" src="https://github.com/user-attachments/assets/4740239a-fc17-4629-b2df-a8eeee7731e4" />

We should pass AA, which requires at least 24px or a spacing equivalent.

As a patch, for now, I have added this in the Customizer on the 2026 site:

```
.wpad-attendees .attendee-social a {
    min-width: 24px;
    min-height: 24px;
    margin: 5px;
    text-align: center;
}
```

This ensures that is passes for AA on teh current year site.

<img width="355" height="185" alt="Image" src="https://github.com/user-attachments/assets/88c5e6cd-31b5-4168-9398-6408db12bab5" />

We should make a permanent fix for all sites in the source code before 2027.
